src/components/Metrics.js

```diff
@@ -43,27 +68,42 @@ const Metrics = React.memo(function Metrics({
               cursor: pointer;
             `}
           />
-          <TokenPrice />
+          <TokenPrice currency={currency} />
         </div>
         <div>
           <TokenBalance
             label="Common Pool"
             value={commonPool}
             token={requestToken}
+            currency={currency}
           />
         </div>
         <div>
           <TokenBalance
             label="Token Supply"
             value={totalSupply}
             token={stakeToken}
+            currency={currency}
           />
         </div>
         <div>
           <TokenBalance
             label="Active"
             value={totalActiveTokens}
             token={stakeToken}
+            currency={currency}
+          />
+        </div>
+        <div>
+          <DropDown
+            header="Type"
+            placeholder="USD"
+            selected={currencyIndex}
+            onChange={handleCurrencyChange}
+            css={`
```

Comment:
we always try to place the css prop as the last one of the props, because sometimes is a big one